deploy/kubernetes/scripts/install

```diff
@@ -123,6 +131,16 @@ if [ "$MINIKUBE_RESET" = "1" ]; then
     (minikube delete || true) &>/dev/null
 
     minikube start --memory 4096
+
+    echo '*******************************'
+    echo '***  Initializing minikube  ***'
+    echo '*******************************'
+
+    # some strange race condition with minikube where it isn't
+    # fully initialized and creating resources can fail. So
+    # we wait for 30 seconds. Ick.
+
+    sleep 30
```

Comment:
Weird one, but this started happening with the latest K8s release. I think it loads the statefulset extension afterwards, thus without the `sleep 30` loading Cassandra fails. Kinda hacky but that's par for the course with this script.